app/views/region/facts/_governance_types.html.erb

```diff
@@ -0,0 +1,26 @@
+<section class="fact">
+  <h2 class="fact__header">Governance types<i class="fact__external-link fa fa-external-link-square"></i></h2>
+  <div class="fact__body">
+    <table class="table js-sortable-table">
+      <thead>
+        <tr class="table__header">
+          <th class="table__column table__column--with-sorting">Name</th>
+          <th class="table__column table__column--with-sorting">Governance type</th>
+          <th class="table__column table__column--with-sorting">Count</th>
+          <th class="table__column table__column--with-sorting">%</th>
+        </tr>
+      </thead>
+      <tbody>
+        <% @country.protected_areas_per_governance.each do |row| %>
+          <tr class="table__row">
+            <td class="table__label table__label--link"><%= link_to row["governance_name"], search_url(country: @country.name, governance: row["governance_name"], main: :country) %></td>
+            <td class="table__value"><%= row["governance_type"] %></td>
+            <td class="table__value"><%= row["count"] %></td>
+            <td class="table__value"><%= row["percentage"].to_f.round(2) %></td>
+          </tr>
+        <% end %>
+        </tr>
+      </tbody>
+    </table>
+  </div>
+</section>
```

Comment:
If you used a shared partial I guess you can remove this file.

Reply:
Hmm, thanks @Levia I deleted that file but it looks like I didn't remove correctly using git. It doesn't appear in my git repo, or when I do `git status`

Reply:
Uhm, it probably should appear in git status in a different section; so you then need to do git add for the deleted file as well. Don't worry you can't manage to delete it from the repo, I can do it for you.